packages/react/src/tag-group/tag-group.stories.tsx

```diff
@@ -0,0 +1,171 @@
+import { Calendar, House } from '@obosbbl/grunnmuren-icons-react';
+import type { Meta, StoryObj } from '@storybook/react';
+import { useState } from 'react';
+import { Label, type Selection } from 'react-aria-components';
+
+import { Description } from '../label';
+import {
+  UNSAFE_Tag as Tag,
+  UNSAFE_TagGroup as TagGroup,
+  UNSAFE_TagList as TagList,
+} from './tag-group';
+
+const meta: Meta<typeof TagGroup> = {
+  title: 'TagGroup',
+  component: TagGroup,
+  parameters: {
+    layout: 'fullscreen',
+  },
+};
+
+export default meta;
+
+type Story = StoryObj<typeof TagGroup>;
+
+export const Default: Story = {
+  args: {
+    selectionMode: 'single',
+  },
+  render: (props) => (
+    <div className="p-6">
+      <TagGroup {...props}>
```

Comment:
Kan vi passe på at alle eksemplene i storybooken er universelt utformet, med aria-label for de eksemplene som ikke har det? Nå kommer det warnings i console pga manglende label, aria-label eller aria-labelledby